server/routerlicious/packages/lambdas/src/deli/lambda.ts

```diff
@@ -1782,6 +1793,27 @@ export class DeliLambda extends TypedEventEmitter<IDeliLambdaEvents> implements
 		}
 	}
 
+	private setApiCounterTimer() {
+		if (!this.serviceConfiguration.deliCheckpointMetricInterval) {
```

Comment:
The service configuration object contains properties for `deli` and `scribe` specific configs. 
Can you move these properties into `IDeliServerConfiguration` /  `IScribeServerConfiguration`
https://github.com/microsoft/FluidFramework/blob/2a65abefb2dd4feaa122420be0ff05b44dd6f16b/server/routerlicious/packages/services-core/src/configuration.ts#L164C1-L170C37